client/ayon_houdini/plugins/publish/extract_render.py

```diff
@@ -75,13 +76,14 @@ def process(self, instance):
                 all_frames.extend(value)
         # Check missing frames.
         # Frames won't exist if user cancels the render.
-        missing_frames = [
+        missing_frames = "\n\n - ".join(
```

Comment:
Why do we need two new lines for each individual line, that seems quite overkill?

Reply:
Yes, unfortunately, we need two new lines in order to get one new line in the publish report.
![image](https://github.com/user-attachments/assets/6402839b-582f-42a1-8377-27236f40d1c6)

Reply:
@iLLiCiTiT this feels fishy, right? It should just be fine with a single new line?

Also, I really think we should improve how these files are listed here - maybe we're better off actually printing them as collections? (if they are detected as collections).

Reply:
> Also, I really think we should improve how these files are listed here - maybe we're better off actually printing them as collections? (if they are detected as collections).

`all_frames` object at this point of execution is a flat list of files.

Reply:
I meant something like this:
```python
collections, remainder = clique.assemble(files)
collections.extend(remainder)
collections_str = "\n".join(f"- {sequence}" for sequence in collections)
```
So that anything that is considered a sequence would just be reported as a single sequence.

Reply:
https://github.com/ynput/ayon-houdini/pull/193/commits/33568f8bfcd862f816b938e5d38f3b93fe80b11c
![image](https://github.com/user-attachments/assets/6b1fb880-3455-4d5c-a467-48e4ce22baf8)

Reply:
The way you implemented it - it would never report the `remainder` which is technically incorrect because those would still be frames it was expecting and had missing. It may happen if for whatever reason `clique` is unable to detect the sequence, e.g. with the current call it won't detect a sequence if it's a single frame (which is fine because we don't need to list those as a sequence). However, we do still want to list is.

That's why in my psuedocode I added them to the list of collections (which is weird due to the variable name of course) but this might make more sense:
```
missing_file_labels = [str(collection) for collection in collections]
missing_file_labels.extend(remainder)
```
And then reporting these file labels.

Reply:
> It may happen if for whatever reason `clique` is unable to detect the sequence, e.g. with the current call it won't detect a sequence if it's a single frame

For this particular case, I've used `minimum_items=1`
```python
missing_frames , _ = clique.assemble(
            missing_frames,
            patterns=[clique.PATTERNS["frames"]],
            minimum_items=1
        )
```

Reply:
> @iLLiCiTiT this feels fishy, right? It should just be fine with a single new line?

It should. How it does look like with single new line then?

Reply:
@iLLiCiTiT 
Here's the output of using only one `\n` and no `-` in `detail`
![image](https://github.com/user-attachments/assets/93e12c12-2d03-456c-b743-e702d88c4615)
The output of combining `\n` and `-` in `detail`
![image](https://github.com/user-attachments/assets/caf91783-136e-4f0e-938d-c834adafcba7)
The output of using two `\n`
![image](https://github.com/user-attachments/assets/81c7b30f-cb26-49c1-b81c-72956381d235)

Reply:
Can you please create issue in ayon-core then?

Reply:
https://github.com/ynput/ayon-core/issues/1062